spec/MC_ffg.tla

```diff
@@ -66,8 +71,12 @@ IsValidBlock(block, node_state) ==
     /\ block.slot >= 0
     /\ block.slot <= MAX_SLOT
     /\ \A voteMsg \in block.votes: IsValidSigedVoteMessage(voteMsg, node_state)
-    \* The set of messages increases monotonically
-    /\ get_block_from_hash(block.parent_hash, node_state).votes \subseteq block.votes
+    /\  LET parent == get_block_from_hash(block.parent_hash, node_state)
+        IN
+        \* Parent has lower slot #
+        /\ parent.slot < block.slot
+        \* The set of messages increases monotonically
+        /\ parent.votes \subseteq block.votes
```

Comment:
I thought `votes` should hold the set of validators that votes for the block. Why should these sets increase monotonically? Is this condition somewhere in the python code or in the paper?

Reply:
I think I assumed these would have been all votes cast by the time the block was constructed, but your interpretation probably makes more sense.

Reply:
@konnov interpretations is basically correct, except that the votes included in a block B are not necessarily for block B. They could be for any block. However, `votes` doest not keep accumulating votes.